src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1 +1,20 @@
-export const TodoInfo = () => {};
+import React from 'react';
+import cn from 'classnames';
+import { Todos } from '../../types/Todos';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todos;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  return (
+    <article
+      data-id={todo.id}
+      className={cn('TodoInfo', { 'TodoInfo--completed': todo.completed })}
+    >
+      <h2 className="TodoInfo__title">{todo.title}</h2>
+      <UserInfo user={todo.user} />
```

Comment:
let's handle this case when user is null, UserInfo doesn't need to know that user can not exist, his goal is to render the information